tools/snvphyl-tools/find_repeats/find-repeats.xml

```diff
@@ -1,14 +1,11 @@
-<tool id="findrepeat" name="Find Repeats" version ="1.8.0">
+<tool id="findrepeat" name="Find Repeats" version ="1.8.2">
   <description>Identify repeat elements using Mummer</description>
   <requirements>
-    <requirement type="package" version="1.8">snvphyl</requirement>
+    <requirement type="package" version="1.8.2">snvphyl-tools</requirement>
     <requirement type="package" version="3.23">mummer</requirement>
```

Comment:
We cannot since at the moment there is issue of running nucmer because of 'missing' perl libraries. It is a known issue where perl libraries are placed in different directories if multiple perl exist in the same conda environment. So for this iteration, would like to keep mummer package in requirements till I can address this later.